### ISSUE TITLE
Honkbot can't be hit my wideswing

### ISSUE THREAD

**Issue by EdenTheLiznerd** [space-wizards/space-station-14#26693]
## Description

Honkbot is unable to be hit by any kind of wide-swing it seems, this makes dealing with honkbots an actual headache for any and all security players, they're worse than they already were

**Reproduction**

Try to wideswing a honkbot, nothing procs

**Comment by EdenTheLiznerd** [comment]
Yeah and your old repo had no issues compared to the fresh one so this behavior on honkbots specifically is really weird

**Comment by deltanedas** [comment]
honkbot survival mechanism, feature
